### PR TITLE
Updated deprecated links in integrate/slash_commands _index.md

### PR DESCRIPTION
**Pull request by Rutam21**
#### Summary

Updated deprecated links in integrate/slash_commands _index.md

#### Ticket Link

NA

### CODE REVIEW

**Comment by Rutam21** [comment]
@cwarnermm Please review and add the necessary labels for Docathon.

File: site/content/integrate/slash-commands/_index.md [hunk 1]
```diff
@@ -72,19 +72,19 @@ Delayed responses are useful when the action takes more than three seconds to pe
 - Retrieval of data from external third-party services, where the response time may take longer than three seconds.
 - Report generation, batch processing or other long-running processes that take longer than three seconds to respond.
 
-Any requests that are made to the response URL should either be a plain text or JSON-encoded body. The JSON-encoded message supports both [Markdown formatting](https://docs.mattermost.com/help/messaging/formatting-text.html) and [message attachments](https://docs.mattermost.com/developer/message-attachments.html).
+Any requests that are made to the response URL should either be a plain text or JSON-encoded body. The JSON-encoded message supports both [Markdown formatting](https://docs.mattermost.com/messaging/formatting-text.html#formatting-text) and [message attachments](https://docs.mattermost.com/developer/message-attachments.html).
```
Comment (justinegeffen) [inline site/content/integrate/slash-commands/_index.md:75]:
```suggestion
Any requests that are made to the response URL should either be a plain text or JSON-encoded body. The JSON-encoded message supports both [Markdown formatting](https://docs.mattermost.com/messaging/formatting-text.html#formatting-text) and [message attachments]({{< ref "admin-message-attachments" >}}).
```

File: site/content/integrate/slash-commands/_index.md [hunk 2]
```diff
@@ -72,19 +72,19 @@ Delayed responses are useful when the action takes more than three seconds to pe
...
 | channel\_id | Overrides the channel to which the message gets posted.<br> Defaults to the channel in which the command was issued. | No |
-| icon\_url | Overrides the profile picture the message posts with.<br> Defaults to the URL set during webhook creation or the webhook creator's profile picture if the former was not set.<br> Must be enabled [in the configuration](https://docs.mattermost.com/administration/config-settings.html#enable-integrations-to-override-profile-picture-icons). | No |
+| icon\_url | Overrides the profile picture the message posts with.<br> Defaults to the URL set during webhook creation or the webhook creator's profile picture if the former was not set.<br> Must be enabled [in the configuration](https://docs.mattermost.com/configure/configuration-settings.html#enable-integrations-to-override-profile-picture-icons). | No |
 | goto\_location | A URL to redirect the user to. Supports many protocols, including `http://`, `https://`, `ftp://`, `ssh://` and `mailto://`.| No |
 | attachments | [Message attachments](https://docs.mattermost.com/developer/message-attachments.html) used for richer formatting options. | If `text` is not set, yes |
```
Comment (justinegeffen) [inline site/content/integrate/slash-commands/_index.md:89]:
```suggestion
| attachments | [Message attachments]({{< ref "admin-message-attachments" >}}) used for richer formatting options. | If `text` is not set, yes |
```

File: site/content/integrate/slash-commands/_index.md [hunk 3]
```diff
@@ -162,7 +162,7 @@ Reply immediately with an `ephemeral` message to confirm response of the command
 
 By default, Mattermost prohibits outgoing connections that resolve to certain common IP ranges, including the loopback (`127.0.0.0/8`) and various private-use subnets.
 
-During development, you may override this behaviour by setting `ServiceSettings.AllowedUntrustedInternalConnections` to `"127.0.0.0/8"` in your `config.json` or via the System Console's Advanced > Developer page. See the [admin guide's notes](https://docs.mattermost.com/administration/config-settings.html#allow-untrusted-internal-connections-to) for more details.
+During development, you may override this behaviour by setting `ServiceSettings.AllowedUntrustedInternalConnections` to `"127.0.0.0/8"` in your `config.json` or via the System Console's Advanced > Developer page. See the [admin guide's notes](https://docs.mattermost.com/configure/configuration-settings.html#allow-untrusted-internal-connections-to) for more details.
```
Comment (justinegeffen) [inline site/content/integrate/slash-commands/_index.md:165]:
```suggestion
During development, you may override this behavior by setting `ServiceSettings.AllowedUntrustedInternalConnections` to `"127.0.0.0/8"` in your `config.json` or via **System Console > Advanced > Developer**. See the [admin guide's notes](https://docs.mattermost.com/configure/configuration-settings.html#allow-untrusted-internal-connections-to) for more details.
```

**Review by justinegeffen — COMMENTED** [review]
Some suggestions have been added, specifically around links (the developer section of the Mattermost docs is no longer in use) and a typo. :)

**Comment by Rutam21** [comment]
@justinegeffen Suggested changes have been applied. Please re-review.

**Review by justinegeffen — APPROVED** [review]
Thank you for reviewing and applying the changes! :)

**Review by cwarnermm — APPROVED** [review]
Thank you for your docs contribution!

**Pull request merged**